site/index.html

```diff
@@ -76,43 +61,54 @@ <h3>Sensible Scripting</h3>
       </div>
 
       <div class="promo_table_cell">
-          <h3>Web Based Configuration</h3>
+          <h3>Reads Your Mind</h3>
 
           <div class="promo_image">
-              <a target="_blank" href="assets/img/screenshots/web_config.png">
-                  <img alt="Fish's web config tool, on the prompt selection tab" src="assets/img/screenshots/web_config.png">
+              <a target="_blank" href="assets/img/screenshots/autosuggestion.png">
+                  <img alt="fish shows suggestions based on files, completions and your history. In this case it offers to turn 'ssh -' into 'ssh -l demo SomeLongHost@SomeLongDomainIAlwaysMisspell.com', and you can accept it with right arrow or control-F" src="assets/img/screenshots/autosuggestion.png">
               </a>
           </div>
 
-          For those lucky few with a graphical computer, you can set your colors and view functions, variables, and
-          history all from a web page.
+          fish suggests commands as you type based on history and completions, just like a web browser.
+          Watch out, Netscape Navigator 4.0!
       </div>
 
       <div class="promo_table_cell promo_table_left">
-          <h3>Man Page Completions</h3>
-          
+          <h3>Abbreviations</h3>
```

Comment:
Good call to remove "man page completions". In future we can mention completions in another way since that's one of fish's strengths